CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift

```diff
@@ -0,0 +1,241 @@
+//
+
+import UIKit
+
+class ErrorScreenViewController: UIViewController {
+
+    enum ErrorType {
+        case connection
+        case uploadIssue
+        case serverError
+        case authentication
+        case unexpected
+    }
+    private var giniConfiguration: GiniConfiguration
+    lazy var errorHeader: NoResultHeader = {
+        if let header = NoResultHeader().loadNib() as? NoResultHeader {
+            header.headerLabel.adjustsFontForContentSizeCategory = true
+            header.headerLabel.adjustsFontSizeToFitWidth = true
+            header.translatesAutoresizingMaskIntoConstraints = false
+        return header
+        }
+        fatalError("No result header not found")
+    }()
+
+    lazy var buttonsView: ButtonsView = {
+        let view = ButtonsView(
+            firstTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.enterManually",
+                comment: "Enter manually"),
+            secondTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.backToCamera",
+                comment: "Enter manually"))
+        return view
+    }()
+
+    lazy var errorContent: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    let viewModel: BottomButtonsViewModel
+    private let errorType: ErrorType
+    private var buttonsHeightConstraint: NSLayoutConstraint?
+    private var numberOfButtons: Int {
+        return [
+            viewModel.isEnterManuallyHidden(),
+            viewModel.isRetakePressedHidden()
+        ].filter({
+            !$0
+        }).count
+    }
+
+    public init(
+        giniConfiguration: GiniConfiguration,
+        type: ErrorType,
+        viewModel: BottomButtonsViewModel
+    ) {
+        self.giniConfiguration = giniConfiguration
+        self.viewModel = viewModel
+        self.errorType = type
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+
+    func setupView() {
+        title = NSLocalizedStringPreferredFormat(
+            "ginicapture.error.title",
+            comment: "Error screen title")
+        errorHeader.iconImageView.accessibilityLabel = NSLocalizedStringPreferredFormat(
+            "ginicapture.error.title",
+            comment: "Error screen title")
+        errorHeader.headerLabel.text = getErrorTitle(type: errorType)
+        errorHeader.headerLabel.font = giniConfiguration.textStyleFonts[.subheadline]
+        errorHeader.headerLabel.textColor = GiniColor(
+            light: UIColor.GiniCapture.dark1,
+            dark: UIColor.GiniCapture.light1
+        ).uiColor()
+        errorContent.text = getErrorContent(type: errorType)
+        errorContent.textAlignment = .left
```

Comment:
The UILabel's textalignment is .left by default. If you need it somehow, I would put it in the initializer of the view